pkg/domain/infra/tunnel/images.go

```diff
@@ -377,6 +379,10 @@ func (ir *ImageEngine) Build(_ context.Context, containerFiles []string, opts en
 	if err != nil {
 		return nil, err
 	}
+	report.SaveFormat = define.OCIArchive
+	if opts.OutputFormat == buildah.Dockerv2ImageManifest {
```

Comment:
this should use buildah/define for this const to avoid the full buildah import.

Reply:
Fixed